ui.tests/test-module/specs/ruleeditor/ruleEditor.runtime.spec.js

```diff
@@ -142,3 +142,42 @@ describe("Rule editor submission handler runtime", () => {
         }
     });
 })
+
+describe("Rule editor save handler runtime", () => {
+
+    let toggle_array = [];
+
+    before(() => {
+        cy.fetchFeatureToggles().then((response) => {
+            if (response.status === 200) {
+                toggle_array = response.body.enabled;
+            }
+        });
+    });
+
+    const saveRunTime = "content/forms/af/core-components-it/samples/ruleeditor/save/saveruntime.html"
+
+    it("should save formData on button click", () => {
+        if (toggle_array.includes("FT_FORMS-11581")) {
+
+            const saveApiResponse = {
+                'draftId': 'ABC'
+            };
+            // Rule when button is clicked then save call should trigger
+            cy.intercept('POST' , '**/adobe/forms/af/save/*', saveApiResponse).as('afSave');
```

Comment:
Why do we need mock for e2e test case, can you please test the actual behavior ?

Reply:
OOTB draft gets stored in Azure, and if we provide azure configuration for test cases, then that will get polluted over time. So I discussed with Rahul and decided to mock the API.